scripts/pr-infra-compose.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/sh
+
+# This script enables the deployment of a dedicated app instance using the
+# following inputs:
+#   * PR Number: Specify the pull request number for the instance you wish to
+#       run.
+#   * Infrastructure Backup: Provide a backup of the infrastructure to load
+#       into the app.
+#
+# Key Benefits:
+#   * Isolated Environment: The new instance operates independently of the
+#       development environment, allowing both to run in parallel without
+#       interference.
+#   * Faster Reviews: The script pulls Docker images generated by CI,
+#       eliminating the need to rebuild everything from scratch, which
+#       accelerates the review process.
+
+
+set -e
+
+# Usage
+
+if  {
+        [ "$#" -eq 3 ] &&
+        [ "$2" != "up-and-load-backup" ]
+    } ||
+    {
+        [ "$#" -eq 2 ] &&
+        {   [ "$2" != "up" ] &&
+            [ "$2" != "down" ] &&
+            [ "$2" != "down-and-clean" ]
+        }
+    }; then
+
+    echo "Usage: $0 pr-number (up | up-and-load-backup [osrd.backup] | down | down-and-clean )" >&2
+    exit 1
+
+fi
+
+# Change ports in needed files
+export PR_NB="pr-$1"
+
+if [ "$2" = "up" ] || [ "$2" = "up-and-load-backup" ]; then
+
+    # Load the docker instance
+    docker compose \
+        -p "osrd-pr-tests" \
+        -f "docker/docker-compose.pr-tests.yml" \
+        up -d
+
+    if [ "$2" = "up-and-load-backup" ]; then
+
+        # Load backup
+        export PR_TEST=1
+        ./scripts/load-backup.sh "$3"
+        export PR_TEST=0
```

Comment:
Why are you doing this? You could leave it set

Reply:
my bad, I thought it was exported all the way up to the current shell